pkg/lang/javascript/nodejs.go

```diff
@@ -37,6 +38,22 @@ func (f *PackageFile) Path() string {
 	return f.path
 }
 
+func (f *PackageFile) MergeDeps(other *PackageFile) {
+	for dep, version := range other.Content.Dependencies {
+		currentVersion, ok := f.Content.Dependencies[dep]
+		if ok {
+			if currentVersion != version {
+				zap.S().Warnf(`Found conflicting dependencies in package.json.
```

Comment:
Nice, I like this approach and the logging!

Only thought is, do we today have any other log messages that are multiline within the message itself? An alternate way to do this would be to have the message just be `Found conflicting dependencies in package.json`, and model the original/current/used as a zap field.

Reply:
yeah let me see how that looks compared to this